src/vue-testing-library.js

```diff
@@ -47,7 +47,7 @@ function render(
   //   additionalOptions = configurationCb(router)
   // }
 
-  const wrapper = mount(
+  let wrapper = mount(
```

Comment:
No need to make it mutable, right? Could we just create a local variable inside of rerender at leave this as a `const`?

Reply:
Yeah see, that's the hard part. Because the properties in the returned object point to `wrapper`, that binding is actually what needs to be updated. If `wrapper` is `const`, then it can't be reassigned. So when `rerender` is called, the original wrapper is destroyed, a new component is mounted, and the test looks fine. But any consecutive times that `rerender` is called in the same test (which is a realistic scenario), the the code will be working with a reference to the oldest wrapper, not the new one. So nothing new will be destroyed, and the DOM will get polluted very quickly. Moreover, users will be unable to take advantage of all utility functions that use `wrapper` (for the aforementioned reason).

Theoretically, there _might_ be some things we could try to keep it `const`? But in the interest of keeping things simple, I think it's best to use `let`. But as long as we're careful about how we use `wrapper`, we should be okay.

I promise I did try using `const` before my first commit, though! 😅

Reply:
Due to what I just described, I updated the first test to call `rerender` twice. This should help avoid unexpected bugs, like in the case of the `updateProps` test on `master`.

Since I didn't submit a code change for this one, I'll leave the resolution up to you. If you think of any simple workarounds let me know. I'll keep thinking while the PR is open.